src/domain/community/virtualContributorAdmin/components/ExternalConfig.tsx

```
@@ -0,0 +1,139 @@
+    [aiPersonaService?.id]
+  );
+
+  const validationSchema = yup.object().shape({
+    apiKey: yup.string().notRequired(),
+    asissantId: isAssistantFieldAvailable ? yup.string().required() : yup.string().notRequired(),
+    model: yup.mixed<OpenAiModel>().oneOf(Object.values(OpenAiModel)).required(),
+  });
+
+  const handleSubmit = () => {
+    if (!externalConfig.apiKey || aiPersonaService?.externalConfig?.apiKey === externalConfig.apiKey) {
+      delete externalConfig.apiKey;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)